app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -72,7 +74,10 @@ input[type="file"] {
 }
 
 select {
+  -webkit-appearance: none;
```

Comment:
Properties should be ordered -moz-appearance, -webkit-appearance, appearance, background, line-height, margin-bottom